keyboards/handwired/aranck/keymaps/turkishish/keymap.c

```diff
@@ -0,0 +1,239 @@
+/* Copyright 2019 Arda Kilicdagi
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+enum aranck_layers {
+  _QWERTY,
+  _LOWER,
+  _RAISE,
+  _ADJUST,
+  _SPACE,
+  _SODA,
+  _NUMPAD
+};
+
+//Tap Dance Declarations
+enum {
+  TD_SHIFT_CAPS = 0,
+};
+
+
```

Comment:
If you decide to go with the non-Tap Dance implemenation, Lines 29-34 should be deleted.